modules/mobile/app/controllers/mobile/v0/users_controller.rb

```diff
@@ -4,7 +4,7 @@ module Mobile
   module V0
     class UsersController < ApplicationController
       after_action :pre_cache_resources, only: :show
-      after_action :link_user_with_vet360, only: :show, if: -> { @current_user.vet360_id.blank? }
+      after_action :handle_vet360, only: :show
```

Comment:
I assume this will raise errors because of the name mismatch.

Reply:
Thanks for catching that